### DISCUSSION TITLE
Implenting with article types

### DISCUSSION THREAD

**Discussion by Fra1975** [asreview/asreview#1013] [Ideas]
Hello. 
I wondered if the system could be implemented and allow filter different types of paper (e.g. Review of the literature; Case report; Case series; Letter to the editor; Consensus; Guidelines; Non-english articles.; etc.)

Many thanks
Francesco

**Comment by Rensvandeschoot** [comment]
@Fra1975, we do not have a feature to filter types of texts because the ASReview pipeline is made generic and can process any type of textual data, including [newspapers ](https://github.com/asreview/asreview/discussions/560)and [emails](https://github.com/asreview-ftm-hackathon/ASReview-plugin-FTM). As long as you import a file with rows being the records and columns with the text, the file can be processed. If you want to filter on specific topics, you can do so in reference manager software and import different datasets into ASReview. Then, you can screen the separate files and merge these afterward. Should be relatively easy.
